### ISSUE TITLE
fix tests.foreman.api.test_bookmarks.BookmarkTestCase failing due to "docker_registries is not a valid controller".

### ISSUE THREAD

**Issue by jameerpathan111** [SatelliteQE/robottelo#6923]

**Comment by jameerpathan111** [comment]
```
2019-05-11 12:26:04 - nailgun.client - WARNING - Received HTTP 422 response: {
  "error": {"id":null,"errors":{"controller":["docker_registries is not a valid controller"]},"full_messages":["Controller docker_registries is not a valid controller"]}
}
```

**Comment by jameerpathan111** [comment]
fixed in #6924
